client/state/notices/middleware.js

```diff
@@ -141,6 +143,18 @@ export const onPublicizeConnectionUpdateFailure = ( dispatch, { error } ) => dis
 	} ) )
 );
 
+export function onSiteSettingsSaveFailed( dispatch, action ) {
+	let text;
+	switch ( action.error.error ) {
+		case 'invalid_ip':
+			text = translate( 'One of your IP Addresses was invalid. Please try again.' );
```

Comment:
This is probably technically accurate but also very confusing from a user perspective I would imagine. "One of my IP addresses was invalid? wat?" since the user didn't actually do anything related to setting or using IP addresses. Actually, as a developer looking at this code I have no idea what the error means or why/when it should appear. Maybe we should hide this from users and try to handle it in the code or simply mask it as "Something unexpected happened and we couldn't save your settings."